artscraper/artist_works_metadata.py

```diff
@@ -58,3 +60,38 @@ def get_artist_works(artist_link, min_wait_time):
     list_links = [element.get_attribute('href') for element in elements]
 
     return list_links
+
+def get_artist_description(artist_link):
+
+    '''
+    Parameters
+    ----------
+    artist_link : Web address of the artist's Google Arts & Culture page
+
+    Returns
+    -------
+    description : Description of artist (lead section of Wikipedia article)
```

Comment:
Can you annotate the return type? Is it a string/dictionary?

Reply:
It's a string; I will clarify that in the docstring.